recipes-kernel/linux/files/iot2050_defconfig_base

```diff
@@ -691,3 +691,8 @@ CONFIG_SAMPLES=y
 CONFIG_SAMPLE_RPMSG_CLIENT=m
 CONFIG_MEMTEST=y
 CONFIG_TI_DAVINCI_MDIO=y
+CONFIG_SERIAL_AMBA_PL011=y
+CONFIG_SERIAL_AMBA_PL011_CONSOLE=y
+CONFIG_SCSI_VIRTIO=y
+CONFIG_DRM_VIRTIO_GPU=y
+CONFIG_VIRTIO_INPUT=y
```

Comment:
Missing newline at end of file.

And this defconfig extension should probably go into the extra set as it is not need for the IOT2050 base.

Reply:
..and that as an own commit.

Reply:
Done.